Index: xquery/src/org/exist/atomic/atom.xql
===================================================================
--- xquery/src/org/exist/atomic/atom.xql	(revision 105)
+++ xquery/src/org/exist/atomic/atom.xql	(working copy)
@@ -289,7 +289,7 @@
         httpclient:delete($uri, false(), atom:get-headers($credentials))
 };
 
-declare function atom:module-imports() as xs:string {
+declare function atom:module-imports() {
     for $module in doc("/db/atom/configuration.xml")//extensions/module 
     return
         util:import-module(xs:anyURI($module/@namespace), $module/@prefix, xs:anyURI($module/@path))
