diff -Nru a/db/wiki/util/latestFeeds.xql b/db/wiki/util/latestFeeds.xql
--- a/db/wiki/util/latestFeeds.xql	2009-05-12 20:06:04.000000000 +0200
+++ b/db/wiki/util/latestFeeds.xql	2009-06-24 03:13:54.000000000 +0200
@@ -2,6 +2,7 @@
 
 import module namespace cfg="http://atomic.exist-db.org/xq/config" at "configuration.xql";
 import module namespace atom="http://www.w3.org/2005/Atom" at "atom.xql";
+import module namespace gui='http://www.cnio.es/scombio/xcesc/1.0/xquery/guiManagement' at 'xmldb:exist:///db/XCESC-logic/guiManagement.xqm';
 
 <div class="entry-list" xmlns="http://www.w3.org/1999/xhtml">
 {
@@ -15,9 +16,9 @@
 let $feedPath := if (string-length($feed) eq 0) then $feed else concat('/', $feed)
 let $url :=
     if ($entry/wiki:id) then
-        concat(cfg:get-html-uri(), $feedPath, "/", $entry/wiki:id)
+        concat(cfg:get-html-uri(), gui:get-gui-path() , $feedPath, "/", $entry/wiki:id)
     else
-        concat(cfg:get-html-uri(), $feedPath, "/?id=", $entry/atom:id)
+        concat(cfg:get-html-uri(), gui:get-gui-path() , $feedPath, "/?id=", $entry/atom:id)
 return
     <div class="entry-list-entry">
         <div class="date">
@@ -28,4 +29,4 @@
         </a>
     </div>
 }
-</div>
\ No newline at end of file
+</div>
diff -Nru a/db/wiki/util/mainIndex.xql b/db/wiki/util/mainIndex.xql
--- a/db/wiki/util/mainIndex.xql	2009-05-12 20:06:04.000000000 +0200
+++ b/db/wiki/util/mainIndex.xql	2009-06-24 03:15:23.000000000 +0200
@@ -2,6 +2,7 @@
 
 import module namespace atom="http://www.w3.org/2005/Atom" at "atom.xql";
 import module namespace cfg="http://atomic.exist-db.org/xq/config" at "configuration.xql";
+import module namespace gui='http://www.cnio.es/scombio/xcesc/1.0/xquery/guiManagement' at 'xmldb:exist:///db/XCESC-logic/guiManagement.xqm';
 
 <div class="main-index" xmlns="http://www.w3.org/1999/xhtml">
 {
@@ -25,9 +26,9 @@
             let $feedPath := if (string-length($feed) eq 0) then $feed else concat('/', $feed)
             let $url :=
                 if ($entry/wiki:id) then
-                    concat(cfg:get-html-uri(), $feedPath, "/", $entry/wiki:id)
+                    concat(cfg:get-html-uri(), gui:get-gui-path(), $feedPath, "/", $entry/wiki:id)
                 else
-                    concat(cfg:get-html-uri(), $feedPath, "/?id=", $entry/atom:id)
+                    concat(cfg:get-html-uri(), gui:get-gui-path(), $feedPath, "/?id=", $entry/atom:id)
             return
                 <div class="main-index-entry">
                     <a href="{$url}">
diff -Nru a/db/wiki/util/search.xql b/db/wiki/util/search.xql
--- a/db/wiki/util/search.xql	2009-05-12 20:06:04.000000000 +0200
+++ b/db/wiki/util/search.xql	2009-06-24 03:16:35.000000000 +0200
@@ -9,6 +9,7 @@
 
 import module namespace kwic="http://exist-db.org/xquery/kwic" at "kwic.xql";
 import module namespace ft="http://exist-db.org/xquery/lucene";
+import module namespace gui='http://www.cnio.es/scombio/xcesc/1.0/xquery/guiManagement' at 'xmldb:exist:///db/XCESC-logic/guiManagement.xqm';
 
 declare variable $ws:FIELDS :=
 	<fields>
@@ -28,9 +29,9 @@
 	let $queryPart := concat("dq=", escape-uri($docXPath, true())) 
 	return
     	if ($entry/wiki:id) then
-        	concat(cfg:get-html-uri(), $feedPath, "/", $entry/wiki:id, "?", $queryPart)
+        	concat(cfg:get-html-uri(), gui:get-gui-path(), $feedPath, "/", $entry/wiki:id, "?", $queryPart)
     	else
-        	concat(cfg:get-html-uri(), $feedPath, "/?id=", $entry/atom:id, '&amp;', $queryPart)
+        	concat(cfg:get-html-uri(), gui:get-gui-path(), $feedPath, "/?id=", $entry/atom:id, '&amp;', $queryPart)
 };
 
 (:~
