--- ../weeklyGOPHER-eXist-module/eXist/conf.xml.tmpl	2009-06-12 16:07:51.000000000 +0200
+++ conf.xml.tmpl	2009-06-14 22:40:25.000000000 +0200
@@ -108,8 +108,8 @@
         you have some more memory to waste. If you deal with lots of 
         collections, you can also increase the collectionCacheSize value 
     -->
-    <db-connection cacheSize="48M" collectionCache="24M" database="@database@"
-        files="webapp/WEB-INF/data" pageSize="4096">
+    <db-connection cacheSize="@memory@" collectionCache="24M" database="native"
+        files="@data.dir@" pageSize="4096">
 
         <!--
             Specifies the default permissions for all resources and collections
@@ -243,8 +243,8 @@
 				backup to the same directory.
         -->
         <recovery enabled="yes" group-commit="no" 
-			journal-dir="webapp/WEB-INF/data" size="100M"
-            sync-on-commit="no" force-restart="no"
+			journal-dir="@data.dir@" size="100M"
+            sync-on-commit="yes" force-restart="no"
 			consistency-check="yes"/>
 
         <!-- 
@@ -401,9 +401,7 @@
             collection-specific configuration for a collection.
         -->
         <index>
-            <fulltext attributes="true" default="all">
-                <exclude path="/auth"/>
-            </fulltext>
+            <fulltext attributes="false" default="none"/>
         </index>
     </indexer>
 
@@ -496,7 +494,23 @@
             <parameter name="backup" value="no"/>
         </job>
         -->
+
+	<job type="user" name="weeklyGOPHER" cron-trigger="0 23 * * 4 ?"
+		xquery="/db/XCESC-logic/cronjob.xql"
+	/>
         
+        <job type="system" name="backup"
+            class="org.exist.storage.BackupSystemTask" 
+            cron-trigger="0 0 */6 * * ?">
+            <parameter name="dir" value="@backup.dir@"/>
+            <parameter name="suffix" value=".zip"/>
+            <parameter name="prefix" value="XCESC-backup-"/>
+            <parameter name="collection" value="/db"/>
+            <parameter name="user" value="@admin.user@"/>
+            <parameter name="password" value="@admin.pass@"/>
+            <parameter name="zip-files-max" value="28"/>
+        </job>
+	
         <!-- 
             Automatically creates a copy of the database .dbx files every 2 minutes
             
@@ -624,7 +638,10 @@
         and "string".  Anything else will be treated as type "string".
         
     -->
-    <transformer class="org.apache.xalan.processor.TransformerFactoryImpl"/>
+    <transformer class="net.sf.saxon.TransformerFactoryImpl">
+    	<attribute name="http://saxon.sf.net/feature/version-warning"
+		 value="false" type="boolean"/>
+    </transformer>
 
     <!--
         Settings for XML validation
@@ -638,7 +655,7 @@
             to the parser.   
             
      -->
-    <validation mode="auto">
+    <validation mode="no">
         <!-- 
             Specify the location of one or more catalog files. Catalogs are 
             used to resolve external entities in XML documents.
@@ -769,11 +786,28 @@
                     uri="http://exist-db.org/xquery/xmldiff" />
             <module class="org.exist.xquery.modules.cache.CacheModule"
                 uri="http://exist-db.org/xquery/cache" />
+            -->
 
             <module class="org.exist.xquery.modules.jfreechart.JFreeChartModule"
                     uri="http://exist-db.org/xquery/jfreechart"/> 
+
+			<module uri="http://www.w3.org/2005/Atom"
+				src="resource:org/exist/atomic/atom.xql"/>
+			<module uri="http://atomic.exist-db.org/xq/config"
+				src="resource:org/exist/atomic/configuration.xql"/>
+			<module uri="http://atomic.exist-db.org/xquery/data"
+				src="resource:org/exist/atomic/data.xql"/>
+			<module uri="http://atomic.exist-db.org/xquery/render"
+				src="resource:org/exist/atomic/render.xql"/>
+			<module uri="http://atomic.exist-db.org/x2cw"
+				src="resource:org/exist/atomic/XHTML2CommonWiki.xql"/>
                
-            -->
+			<module class="org.exist.xquery.modules.math.MathModule"
+				uri="http://exist-db.org/xquery/math" />
+			<module class="org.exist.xquery.modules.httpclient.HTTPClientModule"
+				uri="http://exist-db.org/xquery/httpclient" />
+			<module class="org.exist.xquery.modules.mail.MailModule"
+				uri="http://exist-db.org/xquery/mail" />
         </builtin-modules>
     </xquery>
 
