--- server.xml.tmpl	2009-05-27 22:36:57.000000000 +0200
+++ server.xml.tmpl	2009-06-24 04:13:13.000000000 +0200
@@ -8,7 +8,11 @@
 <server xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="schema/server.xsd">
     
     <!-- default http listener -->
-    <listener protocol="http" port="8088"/>
+    <listener protocol="http" port="@port@"/>
+    
+    <context>
+    	<param name="chiba.configfile" value="chiba/chiba.xml"/>
+    </context>
     
     <!-- https listener 
     
@@ -47,18 +51,55 @@
         <param name="form-encoding" value="UTF-8"/>
         <param name="container-encoding" value="UTF-8"/>
     </rest>
-    <servlet enabled="no"
-             context="/atom/*"
-             class="org.exist.atom.http.AtomServlet"/>
-
+	
+	<!--__________________________________-->
+	<!--    Starts configuration block    -->
+	<!--__________________________________-->
+	
+	<!-- Atom Publishing Protocol -->
+	<servlet enabled="yes"
+		context="/atom/*"
+		class="org.exist.atom.http.AtomServlet"/>
+	
+	<!-- Atomic wiki -->
+	<servlet enabled="yes"
+		context="/thumbs"
+		class="org.atomicwiki.servlets.GetThumbnail"/>
+	
+	<!-- XQueryURLRewrite -->
+	<filter enabled="yes"
+		path="/@AtomicVirtual@/*"
+		class="org.exist.http.urlrewrite.XQueryURLRewrite">
+		
+		<param name="xquery" value="xmldb:exist:///db/@AtomicLogic@/dispatcher.xql"/>
+	</filter>
+		
 	<!--
 		Define Server Filters 
 	-->
-	<filter enabled="no" path="/*" class="uk.gov.devonline.www.xforms.XFormsFilter"/>
-
-    <filter enabled="no" path="/*" class="org.exist.http.urlrewrite.XQueryURLRewrite">
-        <param name="xquery" value="controller.xql"/>
-    </filter>
+	<filter enabled="yes" path="/@XForms.dir@/*" class="org.chiba.agent.web.filter.XFormsFilter">
+		<param name="chiba.debug" value="false"/>
+		<param name="chiba.configfile" value="chiba/chiba.xml"/>
+		<param name="chiba.upload" value="@tmpdir@"/>
+		<param name="useragent" value="dojo"/>
+	</filter>
+	
+	<servlet enabled="yes" context="/Flux/*" class="org.directwebremoting.servlet.DwrServlet">
+		<param name="debug" value="true"/>
+	</servlet>
+	
+	<servlet enabled="yes" context="/FluxHelper/*" class="org.chiba.agent.web.flux.FluxHelperServlet">
+	</servlet>
+	
+	<servlet enabled="yes" context="/PlainHtml/*" class="org.chiba.agent.web.servlet.PlainHtmlServlet">
+	</servlet>
+	
+	<servlet enabled="yes" context="/view" class="org.chiba.agent.web.servlet.ViewServlet">
+	</servlet>
+	<!--__________________________________-->
+	<!--    Finish configuration block    -->
+	<!--__________________________________-->
+	
 
     <!--
         Define URL forwardings
@@ -69,10 +110,6 @@
             if you access the server without specifying a path.
         -->
         <!--root destination="/db/admin/admin.xql"/-->
-        <!-- 
-            Other forwardings: all requests to the specified path
-            will be forwarded to the destination.
-        -->
-        <forward path="/admin" destination="/db/admin/admin.xql"/>
+	<root destination="/db/index.html"/>
     </forwarding>
 </server>
