--- ../weeklyGOPHER-eXist-module/eXist/server.xml.tmpl	2009-05-27 22:36:57.000000000 +0200
+++ server.xml.tmpl	2009-06-24 17:40:41.000000000 +0200
@@ -8,7 +8,7 @@
 <server xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="schema/server.xsd">
     
     <!-- default http listener -->
-    <listener protocol="http" port="8088"/>
+    <listener protocol="http" port="@port@"/>
     
     <!-- https listener 
     
@@ -47,18 +47,32 @@
         <param name="form-encoding" value="UTF-8"/>
         <param name="container-encoding" value="UTF-8"/>
     </rest>
-    <servlet enabled="no"
-             context="/atom/*"
-             class="org.exist.atom.http.AtomServlet"/>
-
-	<!--
-		Define Server Filters 
-	-->
-	<filter enabled="no" path="/*" class="uk.gov.devonline.www.xforms.XFormsFilter"/>
-
-    <filter enabled="no" path="/*" class="org.exist.http.urlrewrite.XQueryURLRewrite">
-        <param name="config" value="xmldb:exist:///db/controller-config.xml"/>
-    </filter>
+	
+	<!--__________________________________-->
+	<!--    Starts configuration block    -->
+	<!--__________________________________-->
+	
+	<!-- Atom Publishing Protocol -->
+	<servlet enabled="yes"
+		context="/atom/*"
+		class="org.exist.atom.http.AtomServlet"/>
+	
+	<!-- Atomic wiki -->
+	<servlet enabled="yes"
+		context="/thumbs"
+		class="org.atomicwiki.servlets.GetThumbnail"/>
+	
+	<!-- XQueryURLRewrite -->
+	<filter enabled="yes"
+		path="/*"
+		class="org.exist.http.urlrewrite.XQueryURLRewrite">
+		
+		<param name="config" value="xmldb:exist:///db/@XCESCConfig@/controller-config.xml"/>
+	</filter>
+	<!--__________________________________-->
+	<!--    Finish configuration block    -->
+	<!--__________________________________-->
+	
 
     <!--
         Define URL forwardings
@@ -69,10 +83,6 @@
             if you access the server without specifying a path.
         -->
         <!--root destination="/db/admin/admin.xql"/-->
-        <!-- 
-            Other forwardings: all requests to the specified path
-            will be forwarded to the destination.
-        -->
-        <forward path="/admin" destination="/db/admin/admin.xql"/>
+	<root destination="/db/www/index.html"/>
     </forwarding>
 </server>
